.github/workflows/test.yaml

```diff
@@ -1,5 +1,9 @@
 # Try to get a short workflow name and a job name that start with Python
 # version to make it easier to check the status inside GitHub UI.
+#
+# When using external actions check that the extenral repos are permitted via
+# the GitHub configuration at https://github.com/twisted/twisted/settings/actions
+#
```

Comment:
I don't know what to say about this.

At work, I fork all the actions that I use to have only reviews/approved code executed.

Another suggestion was to pin a specific commit for the external actions.


I went with configuring GitHub to only allow actions from selected repos.

Reply:
In theory you can have a different commit with the same hash... but sneaking it unobserved is harder, than sneaking a security issue in the latest version that is automatically used by Twisted without any review, any time the tag is changed.

No need to block the PR. Just wanted to raise the idea.